solr/solr-ref-guide/modules/configuration-guide/pages/package-manager-internals.adoc

```diff
@@ -394,7 +411,9 @@ Note that the `Version` value is `"2"`, which means the plugin is updated.
 === How to Avoid Automatic Upgrade
 
 The default version used in any collection is always the latest.
-However, setting a per-collection property in `params.json` ensures that the versions are always fixed irrespective of the new versions added.
+However, setting a per-collection property in `params.json` ensures that the collection uses the same
+package version (i.e., version *2.0*), irrespective of any versions larger than *2.0* that may be added to Solr
```

Comment:
I wonder if we should say "later" or "more recent" than, versus "larger"

Reply:
@epugh I thought of that, but felt that it could be misleading - because, for example, the current package version in Solr could be `2.11.0`, and then the user uploads `2.9.0` (which isn't more recent semantically, but is "larger" lexicographically).

So without the `params.json`, the collection will be "upgraded" to use `2.9.0`, - but if the version `2.11.0` is pinned, it will be used regardless of how the Solr compares the version strings.